hs_core/templates/pages/apps.html

```diff
@@ -0,0 +1,127 @@
+{% extends "pages/page.html" %}
+{% load pages_tags mezzanine_tags keyword_tags comment_tags hydroshare_tags %}
+{% block meta_title %}HydroShare Apps{% endblock %}
+
+{% block extra_css %}
+<style>
+  .apps-container {
+      border-top: 1px solid #cdcdcd;
+      border-bottom: 1px solid #cdcdcd;
+      background: #efefef;
+      margin-bottom: 0;
+  }
+
+  .webapp-listing {
+      min-height: 250px;
+      position: relative;
+      background: white;
+      border: 1px solid #cdcdcd;
+      width: 250px;
+      margin: 15px auto 30px auto;
+      padding: 1em;
+  }
+
+  .webapp-listing .app-description {
+      display: none;
+      position: absolute;
+      top: 0px;
+      font-size: 12px;
+      left: 0px;
+      right: 0;
+      bottom: 0;
+      background: white;
+      padding: 29px 9px 17px 9px;
+  }
+
+  .webapp-listing img {
+      max-height: 210px;
+      margin: 0 auto;
+      display: block;
+  }
+
+  .webapp-listing h3 {
+      position: absolute;
+      background: #093e54;
+      width: 100%;
+      bottom: 0;
+      left: 0;
+      margin: 0;
+      padding: .5em;
+      color: #B5E655;
+  }
+
+  .webapp-listing .app-info-toggle {
+      position: absolute;
+      left: 10px;
+      top: 6px;
+      height: 18px;
+      width: 18px;
+      font-size: 13px;
+      background: white;
+      border-radius: 50%;
+      text-align: center;
+      text-decoration: none;
+      box-shadow: 0 0 1px #000;
+  }
+
+  a.webapp-link {
+      color: #B5E655;
+  }
+</style>
```

Comment:
@aphelionz I think it is better to put these styles in a separate style file, either a separate style file, or part of hydroshare_core.css or hydroshare_extends.css.

Reply:
Roger that. I will move these to another css file